exonum/src/encoding/serialize/mod.rs

```diff
@@ -22,15 +22,24 @@ use encoding::Field;
 use messages::MessageWriter;
 use super::Offset;
 
-/// implement exonum serialization\deserialization based on serde `Serialize`\ `Deserialize`
+/// Implements Exonum serialization\deserialization based on Serde `Serialize`\ `Deserialize`.
 ///
-/// Item should implement:
+/// Under this macro, an item should implement the following entities:
```

Comment:
Please replace "entities" with "traits"; use of "entities" in this context is unorthodox.